docs/proposals/06-immutable-etcd-backups.md

```diff
@@ -0,0 +1,258 @@
+   - Refer to the [Getting Started guide](https://github.com/gardener/etcd-backup-restore/blob/master/docs/usage/enabling_immutable_snapshots.md#configure-bucket-level-immutability) for step-by-step instructions on configuring or updating the immutable bucket across different cloud providers.
+   - For AWS S3, for example, you enable Object Lock at bucket creation; for Azure Blob Storage, you configure Immutable Blob Storage at the container scope.
+
+2. **Set `Etcd.spec.backup.store` to Reference This Bucket**  
+   - In the `Etcd` custom resource (CR), specify the `store` configuration (e.g., `spec.backup.store`) to point to the bucket name you configured above.
+
+3. **Provide Valid Credentials in a Kubernetes Secret**  
+   - The `store` section of the `Etcd` CR must reference a `Secret` containing valid credentials.  
+   - Confirm that this secret has the proper permissions to upload and retrieve snapshots from the immutable bucket.  
+   - See the [Getting Started guide](https://github.com/gardener/etcd-druid/blob/master/docs/deployment/getting-started-locally/getting-started-locally.md#setting-up-cloud-provider-object-store-secret) for an example.
+
+4. **(Gardener Only) Credential Rotation**  
+   - **Note**: In Gardener-based setups, credential rotation is automatically handled by `gardener/gardener`. No manual rotation is required.  
+   - In non-Gardener environments, you must manage credential updates and secret rotation yourself.
+
+By following these steps, you will have set up an immutable bucket for storing etcd backups, along with the necessary references in your `Etcd` specification and Kubernetes secret.
+
+### Handling of Hibernated Clusters
+
+When an etcd cluster is hibernated for a period longer than the bucket’s immutability period, backups might become mutable again (depending on the cloud provider, see [Comparison of Storage Provider Properties](#comparison-of-bucket-level-and-object-level-immutability)). This possibility undermines the intended guarantees of immutability and may expose backups to accidental or malicious alterations.
+
+As mentioned in [gardener/etcd-druid#922](https://github.com/gardener/etcd-druid/issues/922), a clear hibernation signal is needed. Since hibernation is not yet supported in `etcd-druid` and is out of scope for this proposal, we only address the method for maintaining immutability.
+
+#### Proposal
+
+To mitigate the risk of backups becoming mutable during extended hibernation under bucket-level immutability, the authors propose the following approach:
+
+1. **Prerequisite: Take a Final Full Snapshot Before Hibernation**  
+   - Before scaling the etcd cluster down to zero replicas, the etcd controller triggers an [on-demand full snapshot](https://github.com/gardener/etcd-druid/blob/master/docs/proposals/05-etcd-operator-tasks.md#trigger-on-demand-fulldelta-snapshot). This ensures that the latest state of the etcd cluster is captured and securely stored before hibernation commences.
```

Comment:
We also discussed that it is possible that the full snapshot taken prior to hibernation gets corrupted. Then you stand a chance to lose data for at least 24 hours. In addition the full snapshot prior to the one that you are about to take will become mutable after some time. So now you get into a situation where the final full snapshot is corrupt and the one prior to that is mutable and therefore susceptible to modifications. Should we mention this gap?

Reply:
Yes, I've added it as limitation.